src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.cs

```diff
@@ -234,6 +234,7 @@ async Task Test(bool expectOutput)
         }
 
         [Fact]
+        [ActiveIssue("Work out why we aren't getting all the expected diagnostics.")]
```

Comment:
Can we add a test that verifies that the false edits (replace source code with exact same source code) cause the source generator to do no work? Essentially verifying that incremental edits are handled correctly in the generator?